charts/kpow/templates/ingress.yaml

```diff
@@ -2,7 +2,7 @@
 {{- $fullName := include "kpow.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
 {{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
Just realized that there is some logic here to discern the kubernetes version already.  Suppose I could add some more logic to keep this backward compatible with kubernetes versions older than 1.19 if we were looking to get messy, or those consumers can point to an older chart version.

Reply:
Thanks @jacob-m-barnes for your contribution.

I'll just have a quick look/think/learn about that backwards compatibility question - not entirely sure how common pre-1.19 kube is?

Reply:
@d-t-w I went ahead and added backward compatibility for 1.19 and below.  Figure it is better safe than sorry.

Reply:
Thanks @jacob-m-barnes much appreciated!